real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/LoggingNode.java

```diff
@@ -179,9 +185,15 @@ private static String formatDateIntoFile(Date d) {
      * @return the status of the operation - true if it succeeded, false if it didn't
      */
     private boolean createNewLogFile() {
-        if (!outputDirectory.exists() || !outputDirectory.isDirectory()) {
-            new RobobuggyLogicNotification("Output directory path isn't a folder!", RobobuggyMessageLevel.EXCEPTION);
-            return false;
+        if (!outputDirectory.exists()) {
+            if (!outputDirectory.mkdirs()) {
```

Comment:
Since mkdirs makes all the necessary parent directories, do we really need to call it twice?